recipes/zulu-openjdk/all/conanfile.py

```diff
@@ -13,33 +13,32 @@ class ZuluOpenJDK(ConanFile):
     homepage = "https://www.azul.com"
     topics = ("java", "jdk", "openjdk")
     package_type = "application"
-    settings = "os", "arch", "compiler", "build_type"
-
-    @property
-    def _settings_build(self):
-        return getattr(self, "settings_build", self.settings)
+    settings = "os", "arch"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```
Keep the original settings.

Reply:
@valgur can you comment as to why we'd keep the settings here?  My thinking is that we don't use a compiler, so we don't need it's settings.

Reply:
@valgur Just wanted to see if I can bring this back to your attention.
the JDK is a binary package - containing executables and a C library.  For those reasons we wouldn't want the package_id to depend on the compiler and build type.